packages/wallet/src/types/wallet_info.ts

```diff
@@ -7,6 +7,12 @@ export enum NetworkType {
   Mainnet = 'mainnet',
   Testnet = 'testnet',
 }
+// Balance response interface
+export interface BalanceResponse {
```

Comment:
This interface marks the beginning of adding complexity to the codebase. Returning amount as balance is sufficient for now.

Please note that in the future, the node manager should remove outdated nodes from the node list. This ensures that the balance reflects the latest block height.